framework/src/org/checkerframework/framework/util/typeinference/TypeArgInferenceUtil.java

```diff
@@ -222,6 +228,19 @@ public static AnnotatedTypeMirror assignedTo(AnnotatedTypeFactory atypeFactory,
         ErrorReporter.errorAbort("AnnotatedTypes.assignedTo: shouldn't be here!");
         return null; // dead code
     }
+
+    private static boolean isArugment(TreePath path, ExpressionTree argumentTree) {
```

Comment:
Rename to `isArgument`.
Some documentation would be nice.
Travis is failing because two `if(` without space.
Otherwise looks good and can be merged.